.github/workflows/continous_integration.yaml

```
@@ -74,9 +74,4 @@ jobs:
           pytest -v --cov=physical_validation physical_validation/tests/
```

```suggestion
          pytest -v --cov=physical_validation --cov-report=xml physical_validation/tests/
```
For some reason I have found on my other repos that I needed to explicitly ask for an xml report be generated in order for codecov to work correctly